src/main/java/com/sequenceiq/cloudbreak/controller/json/InstanceMetaDataJson.java

```diff
@@ -10,11 +10,20 @@
     private String dockerSubnet;
     private String longName;
     private Integer containerCount;
+    private String hostgroup;
```

Comment:
I think this should be called templateGroup (instanceGroup) instead of hostgroup, because it gets created during stack creation when hostGroups are not even posted.